schemas/asset_schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "description": "Root object for the JSON data structure",
+  "properties": {
+    "metadata": {
+      "type": "object",
+      "description": "Metadata about the asset",
+      "properties": {
+        "copyright": {
+          "type": "string",
+          "description": "Copyright information"
+        },
+        "generator": {
+          "type": "string",
+          "description": "The tool or software used to generate the asset"
+        },
+        "openmaterial_version": {
+          "type": "string",
+          "description": "Version of OpenMATERIAL",
+          "pattern": "^\\d+\\.\\d+\\.\\d+$"
+        },
+        "creator": {
+          "type": "string",
+          "description": "Creator of the asset"
+        },
+        "creation_date": {
+          "type": "string",
+          "description": "Creation date of the asset in the format YYYYMMDDTHHMMSSZ",
+          "pattern": "^\\d{8}T\\d{6}Z$"
+        },
+        "asset_version": {
+          "type": "string",
+          "description": "Version of the asset",
+          "pattern": "^\\d+\\.\\d+\\.\\d+$"
+        },
+        "id": {
+          "type": "string",
+          "description": "Unique identifier for the asset",
+          "pattern": "^[a-f0-9]{32}$"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the asset"
+        },
+        "label": {
+          "type": "string",
+          "description": "Label or category of the asset"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the asset"
+        }
+      },
+      "required": [
+        "copyright",
+        "generator",
+        "openmaterial_version",
+        "creator",
+        "creation_date",
+        "asset_version",
+        "id",
+        "name",
+        "label",
+        "description"
+      ]
+    },
+    "material_mapping": {
+      "type": "array",
+      "description": "Mapping of materials used in the asset",
+      "items": {
+        "type": "array",
+        "description": "Array of RGB values and material name",
+        "items": [
+          {
+            "type": "string",
+            "description": "RGB values in the format 'R;G;B;A'",
+            "pattern": "^\\d{1,3};\\d{1,3};\\d{1,3};\\d{1,3}$"
+          },
+          {
+            "type": "string",
+            "description": "Name of the material"
+          }
+        ],
```

Comment:
I would still suggest adding a third entry for a material description at this level.

This would be useful if an asset file is shared without the specific material property file or an existing mapping table is reused for a not yet labeled geometry.

Reply:
I added a description to the mapping table.